addons/dynamishelper/dynamishelper.lua

```diff
@@ -24,408 +24,427 @@
+            window:hide()
+        else
+            window:show()
+        end
+    elseif command == "size" then
+        if options[1] and tonumber(options[1]) then
+            window:size(options[1])
+            config.save(settings, 'all')
+        else
+            printHelp(command,settings.window.text.size)
+        end
+    elseif command == "font" then
+        if options[1] then
+            window:font(options[1])
+            config.save(settings, 'all')
+        else
+            printHelp(command,settings.window.text.font)
+        end
+    elseif command == "posx" then
+        if options[1] then
+            window:pos_x(options[1])
+            config.save(settings, 'all')
+        else
+            printHelp(command,window:pos_x())
+        end
+    elseif command == "posy" then
+        if options[1] then
+            window:pos_y(options[1])
+            config.save(settings, 'all')
+        else
+            printHelp(command,window:pos_y())
+        end
+    elseif command == "opacity" then
+        if options[1] and tonumber(options[1]) then
+            local opacity = math.abs(tonumber(options[1]))
+            if opacity > 100 then opacity = 100 end
+            window:bg_transparency(opacity/100)
+            config.save(settings, 'all')
+        else
+            printHelp(command,window:bg_transparency()*100)
+        end
+    elseif command == "padding" then
+        if options[1] then
+            window:pad(options[1])
+            config.save(settings, 'all')
+        else
+            printHelp(command,window:pad())
+        end
+    elseif command == "bold" then
+        if window:bold() then
+            window:bold(false)
+            config.save(settings, 'all')
+        else
+            window:bold(true)
+            config.save(settings, 'all')
+        end
+    elseif command == "bgcolor" then
+        if options[3] then
+            window:bg_color(options[1],options[2],options[3])
+            config.save(settings, 'all')
+        else
+            printHelp(command,"")
+        end
+    elseif command == "stroke" then
+        if options[1] and tonumber(options[1]) then
+            window:stroke_width(options[1])
+            config.save(settings, 'all')
+        else
+            printHelp(command,window:stroke_width())
+        end
+    elseif command == "stopacity" then
+        if options[1] and tonumber(options[1]) then
+            local stopacity = math.abs(tonumber(options[1]))
+            if stopacity > 100 then stopacity = 100 end
+            window:stroke_transparency(stopacity/100)
+            config.save(settings, 'all')
+        else
+            printHelp(command,window:stroke_transparency()*100)
+        end
+    elseif command == "stcolor" then
+        if options[3] then
+            window:stroke_color(options[1],options[2],options[3])
+            config.save(settings, 'all')
+        else
+            printHelp(command,"")
+        end
+    elseif command == "ll" then
```

Comment:
I would love to remove this command entirely. LL has been deprecated for six years and the code in here is messy and not particularly useful. If you remove it, I will accept the PR, but I'll leave this up to you.

If you do keep it, no need to get the player name again, it is already set in the `player` variable on load and login.